yohane/__main__.py

```diff
@@ -46,7 +46,14 @@
     show_default=True,
     help="Vocals extractor to use. 'None' to disable.",
 )
-def cli(song_file: Path, lyrics_file: TextIOWrapper, vocals_extractor: str):
+@click.option(
+    "-l",
+    "--language",
+    type=click.STRING,
```

Comment:
Maybe find the pyphen list of supported languages and put it in a `click.Choice` option?

Reply:
there's `pyphen.LANGUAGES`
https://doc.courtbouillon.org/pyphen/stable/api_reference.html#pyphen.LANGUAGES

Reply:
Thanks for the ideas.